docs/web-apps/automated-testing/cypress/yaml/v1.md

```diff
@@ -886,6 +886,31 @@ suites:
 
 ---
 
+### `shardGrepEnabled`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+When sharding is configured, and used in conjunction with `@cypress/grep`, it may happen that some spec files are allocated to VMs, just to be skipped by playwright in accordance with the `@cypress/grep` filters, thus wasting VM allocations.
```

Comment:
```suggestion
When sharding is configured, and used in conjunction with `@cypress/grep`, it may happen that some spec files are allocated to VMs, just to be skipped by cypress in accordance with the `@cypress/grep` filters, thus wasting VM allocations.
```